core/server/master/src/main/java/alluxio/master/metastore/rocks/RocksStore.java

```diff
@@ -114,24 +112,37 @@ private void resetDb() throws RocksDBException {
   private void stopDb() {
     if (mDb != null) {
       try {
+        Closer closer = Closer.create();
         // Column handles must be closed before closing the db, or an exception gets thrown.
         mColumnHandles.forEach(handle -> {
-          handle.get().close();
+          ColumnFamilyHandle column = handle.get();
+          closer.register(() -> column.close());
           handle.set(null);
         });
-        mDb.close();
+        // Synchronize to make findbugs happy.
+        closer.register(() -> {
+          synchronized (this) {
+            mDb.close();
+          }
+        });
+        closer.register(() -> {
+          synchronized (this) {
+            mCheckpoint.close();
+          }
+        });
+        closer.close();
```

Comment:
Does using a closer for all of them make sense, given that `mDb.close` will fail if any of the ColumnFamilyHandles fail?

Reply:
good point, removed the closer